src/Axe.Windows.Actions/Actions/SelectAction.cs

```diff
@@ -85,6 +84,7 @@ private set
                 if (_POIElementContext != null)
                 {
                     dma.RemoveElementContext(_POIElementContext.Id);
+                    _POIElementContext.Dispose();
```

Comment:
>_POIElementContext.Dispose(); [](start = 20, length = 29)

This is a behavioral change. Previous code would leave the previous ElementContext undisposed. Any concerns from this?